releasenotes/notes/logs-fix-empty-log-with-docker-container-64129b051c554ad4.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Sometime agent sends empty logs when running with docker container.
```

Comment:
Let's try to give more details if we can in release notes. this is what is being read by a lot of people that do not have much context about the situation.

Suggestion:

```
Empty logs could be appear when collecting Docker logs in addition of the actual container logs. This was due to the way the Agent handles the header Docker adds to the logs.
The process has been changed to make sure that no empty logs are generated.
```

WDYT?